### ISSUE TITLE
Key pressing doesn't work in ASP Test

### ISSUE THREAD

**Issue by AndreyBelym** [DevExpress/testcafe#591]
`act.press` doesn't work in `ASPxGridViewDemos/Accessibility/KeyboardSupport` test. I've found that the issue is contained in #551.
